.rat-excludes

```diff
@@ -30,6 +30,9 @@ spark-env.sh.template
 log4j-defaults.properties
 bootstrap-tooltip.js
 jquery-1.11.1.min.js
+d3.min.js
+dagre-d3.min.js
+graphlib-dot.min.js
```

Comment:
Do we need to add `dagre-d3` and `graphlib-dot` to `LICENSE`? I have already added `d3` in #5533.